specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/machineLearningServices.json

```diff
@@ -5506,6 +5506,26 @@
         }
       }
     },
+    "ComputeTriggerType": {
```

Comment:
Please add description for this type definition here.

Reply:
done